### PR TITLE
int64 attr and ignore slicing over batch dim

### PR DESCRIPTION
**Pull request by ramondoo**

### CODE REVIEW

**Comment by ramondoo** [comment]
fix start end overflow in slice Op; ignore slice in batch dimension.

File: OnnxAttrs.cpp [hunk 1]
```diff
@@ -42,6 +42,10 @@ template<> std::vector<int> OnnxAttrs::get<std::vector<int>>(std::string key) co
   auto attr = this->at(key)->ints();
   return std::vector<int>(attr.begin(), attr.end());
 }
+template<> std::vector<long long> OnnxAttrs::get<std::vector<long long>>(std::string key) const {
```
Comment (kevinch-nv) [inline OnnxAttrs.cpp]: Would prefer `int64_t` over `long long` everywhere in your PR

File: builtin_op_importers.cpp [hunk 2]
```diff
@@ -1703,6 +1704,13 @@ DEFINE_BUILTIN_OP_IMPORTER(Slice) {
   for (size_t i = 0; i < axes.size(); ++i)
   {
     int axis = axes[i];
+    
+    // ignore slicing over batch dim
+    if (axis == 0)
```
Comment (kevinch-nv) [inline builtin_op_importers.cpp]: This case is already handled below in `TRT_CHECK(convert_axis(axis,nbDims)`, unneeded here.

Comment (ramondoo) [reply builtin_op_importers.cpp]: There are cases where onnx model does contain Slice op over batch dim. And the original code will exit with assertion or give incorrect result.  

My case:
image

onnx exported from pytorch 1.01

**Comment by ramondoo** [comment]
long long to int64_t, and still think code for ignoring slicing over batch is useful, please re-consider.

**Comment by m7thon** [comment]
I ran into exactly the same issues (end overflow and slice over batch dim) as ramondoo, while converting a model with slice operation (`x = x[:, :, h1:h2, w1:w2]`) from PyTorch 1.0.1. This PR fixes it. Thanks!

File: builtin_op_importers.cpp (hunk 2 above)
Comment (m7thon) [reply builtin_op_importers.cpp]: `convert_axis` currently sets the axis to `-1` if it was originally `0`, and then fails asserting that `axis >= 0`. Slicing over batch dimension should probably be allowed for slices such as `[:, :, h1:h2, w1:w2]`.

**Pull request closed without merging**